Frontend/CO.CDP.OrganisationApp.Tests/AuthorizationTests.cs

```diff
@@ -0,0 +1,212 @@
+                                    },
+                                    userScopes,
+                                    new Uri("http://foo")
+                                );
+
+        var person = new Person.WebApiClient.Person("a@b.com", "First name", personId, "Last name");
+
+        tenantClient.Setup(client => client.LookupTenantAsync())
+            .ReturnsAsync(
+                new TenantLookup(
+                    new List<UserTenant>() {
+                        new UserTenant(
+                            new Guid(),
+                            "Tenant name",
+                            new List<UserOrganisation> { organisation }
+                        )
+                    },
+                    new UserDetails("a@b.com", "User name", "urn")
+                ));
+
+        services.AddTransient<ITenantClient, TenantClient>(sc => tenantClient.Object);
+
+
+        organisationClient.Setup(client => client.GetOrganisationPersonsAsync(It.IsAny<Guid>()))
+            .ReturnsAsync(
+                new List<Organisation.WebApiClient.Person>
+                {
+                    new Organisation.WebApiClient.Person("a@b.com", "First name", person.Id, "Last name", new List<string>() { OrganisationPersonScopes.Admin, OrganisationPersonScopes.Editor })
+                }
+            );
+
+
+        organisationClient.Setup(client => client.GetOrganisationPersonInvitesAsync(It.IsAny<Guid>()))
+            .ReturnsAsync(
+                new List<PersonInviteModel>
+                {
+                    new PersonInviteModel("a@b.com", "Person invite", personInviteGuid, "Last name", new List<string>() { OrganisationPersonScopes.Admin, OrganisationPersonScopes.Editor })
+                }
+            );
+
+        organisationClient.Setup(client => client.GetOrganisationAsync(testOrganisationId))
+            .ReturnsAsync(
+                new Organisation.WebApiClient.Organisation(
+                    new List<Identifier> { },
+                    new List<Address> { },
+                    new ContactPoint("a@b.com", "Contact", "123", new Uri("http://whatever")),
+                    testOrganisationId,
+                    new Identifier("asd", "asd", "asd", new Uri("http://whatever")),
+                    "Org name",
+                    new List<Organisation.WebApiClient.PartyRole> { Organisation.WebApiClient.PartyRole.Supplier, Organisation.WebApiClient.PartyRole.ProcuringEntity }
+                )
+            );
+
+        services.AddTransient<IOrganisationClient, OrganisationClient>(sc => organisationClient.Object);
+
+        _mockSession.Setup(s => s.Get<Models.UserDetails>(Session.UserDetailsKey))
+            .Returns(new Models.UserDetails() { Email = "a@b.com", UserUrn = "urn", PersonId = person.Id });
+
+        services.AddSingleton(_mockSession.Object);
+
+        services.AddSingleton<IPolicyEvaluator, FakeAuthenticationPolicyEvaluator>();
```

Comment:
As we are mocking API clients here, we don't need to use FakeAuthenticationPolicyEvaluator

Reply:
Removed 👍🏻